ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json

```diff
@@ -0,0 +1,94 @@
+{
+  "resourceType": "ServiceRequest",
+  "id": "ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example",
+  "extension": [
+    {
+      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-Coverage",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-FundingCategory",
+            "code": "nhs",
+            "display": "NHS"
+          }
+        ]
+      }
+    }
+  ],
+  "status": "active",
+  "intent": "reflex-order",
```

Comment:
I think this would be classed as an order rather than a reflex as this is a new request rather than being requested based on the results of a previous test